meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -1,2 +1,5 @@
 # LS1021atwr supports usb-audio & simple soundcard, alsa utils are required for playback/record operations. Also we need to support vfat.
 MACHINE_FEATURES_append_ls1021atwr = " alsa vfat"
+
+# LS1021atwr install rcw into pre-built binaries.
```

Comment:
pre-built binaries could lead to confusion, given our sstate directory is called cached-binaries :)